app/view_models/cost_per_transaction.rb

```diff
@@ -0,0 +1,52 @@
+class CostPerTransaction
+  DELIVERY_PARTNER_BREAKDOWN_METHODS = %w(split_by_call_volume).freeze
```

Comment:
you would think that. However, the purpose of the freeze statement is to make Rubocop happy rather than an explicit attempt to block edits to the constant - not that I am suggesting we should edit the constant.